packages/web-components/fast-foundation/src/tabs/tabs.ts

```diff
@@ -243,10 +254,12 @@ export class Tabs extends FoundationElement {
         });
     }
 
-    private setComponent(): void {
+    private setComponent(supressFocus: boolean = false): void {
         if (this.activeTabIndex !== this.prevActiveTabIndex) {
             this.activeid = this.tabIds[this.activeTabIndex] as string;
-            this.focusTab();
+            if (!supressFocus) {
+                this.focusTab();
+            }
```

Comment:
I think focusing the tab is default behavior - is the intent here to support auto-rotation? If so, should we consider something more semantic so that the intent is clear?